ql/src/java/org/apache/hadoop/hive/ql/exec/NodeUtils.java

```diff
@@ -31,47 +34,60 @@
 
   public static <T> void iterateTask(Collection<Task<?>> tasks, Class<T> clazz, Function<T> function) {
```

Comment:
add a note that this does breadth first traversal of the tasks ?
(earlier it used to do depth first traversal)